### PR TITLE
add social media guide

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: resources/social-media.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# RustBridge Social Media Guide
+Social Media can be a very powerful tool to leverage your event, but also to motivate attendees to share their results! Because it’s so powerful, it’s very helpful and sometimes necessary to set concrete rules for usage by not only organizers and teachers, but also attendees.
+
+
+## Photo Guidelines
+
+This is the part that is necessary. You want to make sure that your event is governed by a photo taking policy, meaning that you want to ask for people’s *consent* before you take their picture. The same thing goes for attendees taking pictures.
+
+Assume that every individual person is **not** ok with having their picture taken until you ask them to. The easiest way to do this is to sort this out at the beginning by asking the crowd of attendees if they’re okay with it. If not everyone is, *don’t* call them out — tell them that it’s okay, and ask the rest of the group to either be careful with or refrain from taking pictures with people in them altogether.
```
Comment (ashleygwilliams) [inline resources/social-media.md]: sometimes people are not happy saying in front of the group that they are uncomfotable, providing a backchannel like a slack can be very useful to allow people to do this. it also may be worth asking in the emails ahead of time so students can share their preference in private.

File: resources/social-media.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
+## Social Media Hashtags
+
+A cool thing to do is to encourage attendees and teachers to post their learnings or other findings under a hashtag on platforms like Twitter or Instagram. This hashtag should be called `#RustBridgeLOCATION` or `#RustBridgeIDENTIFIER`, so like `#RustBridgeBerlin` or `#RustBridgeDC`. It’s also okay to just post under the `#RustBridge` hashtag, or both! Incentivize tagging posts by offering to retweet them with your event’s account, or the official RustBridge account, but be careful that they don’t violate your photo taking rules, as discussed in the previous section.
```
Comment (ashleygwilliams) [inline resources/social-media.md]: you might mention that the students can DM their posts to the appropriate accounts to help facilitate RTing

File: resources/social-media.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+
+## Social Media Accounts
```
Comment (ashleygwilliams) [inline resources/social-media.md]: might be worth having a section about the types of other content the account should and should not share, i.e. always focus on inclusive learning environments, don't talk down other programs or programming languages, strive to always be positive and only RT known accounts

**Review by ashleygwilliams — COMMENTED** [review]
i think this looks great! just a few comments and suggestions :)

**Review by Manishearth — APPROVED** [review]

**Pull request merged**